src/main/java/com/google/sps/servlets/ElectionServlet.java

```diff
@@ -98,13 +119,48 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
                 conn.disconnect();
             }
         }
+        
+        String elections = strBuf.toString(); 
+
+        JSONObject obj = new JSONObject(elections);
+        JSONArray electionData = obj.getJSONArray("elections");
+        int numElections = electionData.length();
+
+        for (int i = 0; i < numElections; ++i) {
+            JSONObject currentElection = electionData.getJSONObject(i);
+            Entity electionEntity = new Entity("Election");
+
+            electionEntity.setProperty("id", currentElection.getLong("id"));
```

Comment:
Hm. Is there a reason why we didn't use this value as the key/id for the Entity?

Reply:
I was thinking of the "id" value of the election entity as the ID provided by the API and a feature of the election (similar to its name), while the ID provided by datastore is generated when the election is added to Datastore. 

Do you think we should change it to the key/id for the Entity given that information?

Reply:
I think it makes sense to use the ID provided by the API as the ID in our datastore.  There's no need to have datastore generate a new one if we can re-use the ID we already have.  Though granted, I haven't checked if the ID from Civics api conforms with the datastore key requirements, so that might be a blocker.

Reply:
I'll check right now!